src/app/Home/sidebar/sidebar.component.css

```
@@ -0,0 +1,24 @@
+/* sidebar.component.css */
+
+.sidebar {
+    width: 200px;
+    height: 100%;
+    background-color: #007bff;
+    color: #fff;
+    padding: 20px;
+    font-weight: bold;
+    font-size: medium;
```

```suggestion
    width: 12.5rem;
    padding: 1.25rem;
    font-weight: 600;
    font-size: 1rem;
```
As far as I know, we should use **rem** instead of **px**, as **rem** is based on root whether **px** will remain always fixed. We should declare the font-weight 600 rather than bold. Same as in font-size we should use **rem** value instead of **medium**.

Another thing, we can keep the colors in variables and can use the variables where we need to use. It will be helpful in need of change the colors from one place only. Otherwise, we have to manually change each one.